build.include

```diff
@@ -14,7 +14,7 @@ IMAGE_TAG="next"
 THEIA_GITHUB_REPO="eclipse-theia/theia"
 THEIA_VERSION="master"
 THEIA_BRANCH="master"
-THEIA_COMMIT_SHA="2eb4319fb459f30ad897e8467878154c6790aaef"
```

Comment:
not the latest sha, as the latest commit brings another breaking changes
I'll look at it tomorrow

Reply:
https://github.com/eclipse/che/issues/19866